userguide/jails.rst

```diff
@@ -636,10 +640,42 @@ The final set of jail properties are contained in the
    |                        |             | the jail to create tun devices.                                                                         |
    +------------------------+-------------+---------------------------------------------------------------------------------------------------------+
 
+
 Click :guilabel:`SAVE` when the desired jail properties have been set.
 New jails are added to the primary list in the :guilabel:`Jails` menu.
 
 
+.. index:: Creating Template Jails
+.. _Creating Template Jails:
+
+Creating Template Jails
+^^^^^^^^^^^^^^^^^^^^^^^
+
+
+Advanced jail creation can also be used to create a Basejail that can
```

Comment:
A lot of repeated "can"s in this paragraph.  Reword to avoid that redundancy. Also, it switches back and forth between a basejail (should it be capitalized?) and a template jail.
"*Template* jails are basejails that can be used as a template to efficiently create jails with the same configuration. These steps create a template jail:"